blaise82/l.m.s-server
# #170868490 Build register books endpoint

#### What does this PR do?
Have the add books endpoint built

#### Description of task to be completed?
Have the following endpoint working with validations
`POST /api/v1/books/`

#### How should this be manually tested?
- Clone this repo to your local repository using `git clone <repo link>`
- Install all project dependencies using `npm i`
- Create a .env file and add your database information(username, password, host)
- run `sequelize db:migrate` and check if the tables are created in your database.
- Test this endpoint using postman `POST /api/v1/books/`, pass the admin token in the headers.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#170868490](https://www.pivotaltracker.com/story/show/170868490)

#### Screenshots (if appropriate)
<img width="1125" alt="Screen Shot 2020-01-29 at 00 04 14" src="https://user-images.githubusercontent.com/53453380/73310744-92ad1680-422d-11ea-9f7d-dc0b449cefb1.png">
<img width="1128" alt="Screen Shot 2020-01-29 at 00 04 40" src="https://user-images.githubusercontent.com/53453380/73310747-9345ad00-422d-11ea-981a-d9a97d591fc7.png">
<img width="1122" alt="Screen Shot 2020-01-29 at 00 05 10" src="https://user-images.githubusercontent.com/53453380/73310748-9345ad00-422d-11ea-8b02-24d710dde153.png">
<img width="1126" alt="Screen Shot 2020-01-29 at 00 05 58" src="https://user-images.githubusercontent.com/53453380/73310749-93de4380-422d-11ea-8a75-46fe6eef4ca7.png">
